docs/recipes/recipe-content-analytics/index.md

```diff
@@ -27,7 +27,7 @@ We have already created a custom `content` entity for you in [Iglu Central](http
 
 Snowplow uses self-describing JSON schemas to structure events and entities so that they can be validated in the pipeline and loaded into tidy tables in the warehouse. You can learn more about these data structures [here](/docs/understanding-tracking-design/understanding-schemas-and-validation/index.md), and about why we take this approach [here](https://snowplowanalytics.com/blog/2020/01/24/re-thinking-the-structure-of-event-data/).
 
-While Try Snowplow and BDP Cloud Preview only ship with a pre-designed set of custom events and entities required for the recipes, Snowplow BDP lets you create an unlimited number of your own via the [Data Structures UI](/docs/understanding-tracking-design/managing-data-structures/index.md) (and API). Custom events and entities will come soon to BDP Cloud.
+While Try Snowplow only ships with a pre-designed set of custom events and entities required for the recipes, Snowplow BDP lets you create an unlimited number of your own via the [Data Structures UI](/docs/understanding-tracking-design/managing-data-structures/index.md) (and API). 
```

Comment:
Should this be pointing to (not yet existing) docs on the schema builder for BDP Cloud?

Reply:
yup, I think so. Gotta discuss that with @elyerry

Reply:
👍